test/sql/storage/compression/simple_compression.test

```diff
@@ -0,0 +1,35 @@
+# name: test/sql/storage/compression/simple_compression.test
+# description: Test storage with multiple compression schemes
+# group: [compression]
+
+# load the DB from disk
+load __TEST_DIR__/test_compression_simple.db
+
+foreach compression bitpacking rle
```

Comment:
You could use `<compression>` here  and add bitpacking to the `<compression>` flag (see `test_sqllogictest.cpp:1519`).

Reply:
done!